internal/ingress/s3sqs/s3sqs.go

```diff
@@ -140,20 +140,15 @@ func (s *Ingress) acknowledge(msg *awssqs.Message) error {
 
 // Ingest downloads an object from S3 and applies a handler to the downloaded
 // payload. Few of these can be executed in parallel.
-func (s *Ingress) ingest(bucket, key string, handler func(v []byte) bool) {
+func (s *Ingress) ingest(bucket, key string, handler func(v string) bool) {
```

Comment:
Can we confirm it's working before landing this? Otherwise it might break existing setups